### DISCUSSION TITLE
excalidraw

### DISCUSSION THREAD

**Discussion by hedysnike** [coollabsio/coolify#3993] [Service Template Requests]
[excalidraw](https://github.com/excalidraw/excalidraw-room)

**Comment by mredodos** [comment]
+1

i have installed excalidraw via docker image coolify option 

is very simple you need install the image and config port inside the settings

**Reply by arnoldmukisa** [reply]
are you able to share more detailed instructions? that would be really helpful

**Reply by mredodos** [reply]
https://hub.docker.com/r/excalidraw/excalidraw 

you need to set 5000:80 as port

you can click on new resource in your server
select docker image add the docker image above

before deploy set the port 

![image](https://github.com/user-attachments/assets/2411ae06-6c3d-433d-b98a-9a9fdac3442c)

**Reply by Corvalan** [reply]
excalidraw/excalidraw Docker image does not currently support collaboration features as of today 20 april 2025.

excalidraw/excalidraw-room Docker image is designed to serve collaboration rooms for Excalidraw, intended to work alongside the excalidraw/excalidraw client. However, the excalidraw/excalidraw-room image is outdated (last updated over a year ago)

**Comment by nktnet1** [comment]
PR:
- https://github.com/coollabsio/coolify/pull/6095

From what I can see, the docker image was updated last month (May 28th 2025)

- https://hub.docker.com/r/excalidraw/excalidraw/tags

    ![docker-excalidraw](https://github.com/user-attachments/assets/ed7b29bb-9fe6-4287-a794-3c4528402c0b)

**Comment by mirus-ua** [comment]
Maybe we can try to adopt the ideas from this repo to work with coolify, WDYT? 
https://github.com/Vin-Ren/excalidraw-collab-docker
